stix2elevator/cli.py

```diff
@@ -170,6 +170,8 @@ def main():
     elevator_args = elevator_parser.parse_args()
 
     initialize_options(elevator_args)
+    print(sys.getrecursionlimit())
```

Comment:
I'm not sure if we want to set the recursion limit in our CLI, but I can see arguments both ways. If we do, maybe we can set it using a command-line option? I don't think we need to print it out though ;-)

Reply:
Maybe add a option on the command line?

Reply:
I  think we can just remove this for now. If we need to, we can add an option later